cmake/OpenCVFindCANN.cmake

```diff
@@ -58,24 +65,44 @@ if(CANN_INSTALL_DIR)
         return()
     endif()
     #  * libopsproto.so
-    set(lib_opsproto "${CANN_INSTALL_DIR}/opp/op_proto/built-in")
+    if (CANN_VERSION_BELOW_6_3_ALPHA002)
+        set(lib_opsproto "${CANN_INSTALL_DIR}/opp/op_proto/built-in/")
+    else()
+        if(EXISTS "${CANN_INSTALL_DIR}/opp/built-in/op_proto/lib/linux")
+            set(lib_opsproto "${CANN_INSTALL_DIR}/opp/built-in/op_proto/lib/linux/${CMAKE_HOST_SYSTEM_PROCESSOR}")
```

Comment:
What about Windows? Forcing Linux dependency is bad idea. You can add `If (UNIX)` and `if (WIN32)` checks.

Reply:
There is no cann version for Windows, and the "linux" is only folder under the path of  "${CANN_INSTALL_DIR}/opp/built-in/op_proto/lib/linux". So I wonder if it's necessary to make a check?

Reply:
Please add `UNIX` check and warning at the very beginning.